Projects/Server/Buffers/STArrayPool.cs

```diff
@@ -15,7 +15,7 @@ namespace Server.Buffers;
  */
 public class STArrayPool<T> : ArrayPool<T>
 {
-    private const int StackArraySize = 8;
+    private const int StackArraySize = 32;
```

Comment:
In .NET 8, this was changed to 32.

TODO: Refactor this to use the newest .net 9 code.